dotCMS/src/main/java/com/dotmarketing/util/ImportExportUtil.java

```diff
@@ -1378,18 +1407,29 @@ private void doXMLFileImport(File f, PrintWriter out, ObjectFilter filter)throws
                                 	}
                                     HibernateUtil.saveWithPrimaryKey(identifier, prop);
                                 }else{
+                                    Logger.debug(this, "Saving the object: " +
+                                                obj.getClass() + ", with the id: " + prop);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal ", with the id: " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)